modules/nextflow/src/main/groovy/nextflow/conda/CondaCache.groovy

```diff
@@ -285,7 +285,8 @@ class CondaCache {
         def cmd
         if( isYamlFilePath(condaEnv) ) {
             final target = isYamlUriPath(condaEnv) ? condaEnv : Escape.path(makeAbsolute(condaEnv))
-            cmd = "${binaryName} env create --prefix ${Escape.path(prefixPath)} --file ${target}"
+            final yesOpt = binaryName == 'micromamba' ? '--yes ' : ''
+            cmd = "${binaryName} env create ${yesOpt}--prefix ${Escape.path(prefixPath)} --file ${target}"
```

Comment:
Let's add a test capturing the command for micromamba

Reply:
Added in last commit